src/ipyradiant/basic_tools/selection_widget.py

```diff
@@ -42,12 +67,33 @@ def _make_selected_things(self):
     @trt.default("add_button")
     def _make_add_button(self):
         return ipyw.Button(
-            description="Add",
+            description="Add -->".center(12),

```

Comment:
This magic number may cause issues if the buttons are made to by dynamically sized, but we will not worry about this for now. Just be aware that this will cause weird styling in some edge cases.